caddyhttp/proxy/upstream.go

```diff
@@ -431,9 +431,10 @@ func parseBlock(c *caddyfile.Dispenser, u *staticUpstream, hasSrv bool) error {
 		}
 		u.downstreamHeaders.Add(header, value)
 	case "transparent":
+		// Note: X-Forwarded-For header is always being appended for proxy connections
```

Comment:
Added a comment to reason about why the **X-Forwarded-For** header is not set here